vdev/src/commands/mod.rs

```diff
@@ -19,7 +19,10 @@ mod test;
     version,
     bin_name = "vdev",
     infer_subcommands = true,
-    disable_help_subcommand = true
+    disable_help_subcommand = true,
+    after_help = r#"Environment variables:
+  $CONTAINER_TOOL  Set the tool used to run containers (Defaults to autodetect)
```

Comment:
Should this also show the possible values: `docker|podman|autodetect`?

Reply:
Maybe? `autodetect` isn't a possible value, though, that's just what happens when the variable is unset.

Reply:
Ah 🤔 it's definitely friendlier to provide the allowed values when it's (essentially) an `Enum`. But either way not too big of a deal imho.